README.md

```diff
@@ -27,7 +27,7 @@ This series of events will help you develop SaaS applications that can:
 | Session | Topic | Date | Audience | Level | Quick Content Summary |
 |---------|-------|------|----------|-------|-----------------------|
 | 1 | [The ISV Modernization Journey](#Session-1---The-ISV-Modernization-Journey) | 5 March 2021 | Devs, CSAs | 300 - 400 | Hands-on walkthrough modernizing a .NET app |
-| 2 | [Modern Architecture Patterns and Modernizing Compute](#Session-2---Modern-Architecture-Patterns-and-Modernizing-Compute) | Week 1 - TBC | | | |
+| 2 | [Modern Architecture Patterns and Modernizing Compute](#Session-2---Modern-Architecture-Patterns-and-Modernizing-Compute) | Week 2 - Tuesday March 5th  | Devs, Architects, Decision Makers  | 300-400 | A session on Archiectural patterns, Cloud Adoption Patterns and compute options with QnA |
```

Comment:
Is the date correct? It appears as Tuesday, 9th of March below?

Reply:
Yes correct , that's what we agreed (@vitoc @GuideVMSFT ) , since the first webinar is on 5th march. Also we decided to keep it on Tuesday instead of Monday

Reply:
That's a good catch, will change it!